internal/pkg/manifest/workload.go

```diff
@@ -15,8 +15,10 @@ import (
 )
 
 var (
-	errUnmarshalBuildOpts = errors.New("can't unmarshal build field into string or compose-style map")
-	errUnmarshalCountOpts = errors.New(`can't unmarshal "count" field to an integer or autoscaling configuration`)
+	errUnmarshalBuildOpts 	= errors.New("can't unmarshal build field into string or compose-style map")
+	errUnmarshalCountOpts 	= errors.New(`can't unmarshal "count" field to an integer or autoscaling configuration`)
+	errUnmarshalEntryPoint 	= errors.New("can't unmarshal entrypoint into string or slice of strings")
+	errUnmarshalCommand 	= errors.New("can't unmarshal command into string or slice of strings")
```

Comment:
nit: would you mind changing the errors in this package from "can't" to "cannot"? 🙏  
Looks like in the `cli` pkg we use "cannot", as well as in the `stack` pkg.